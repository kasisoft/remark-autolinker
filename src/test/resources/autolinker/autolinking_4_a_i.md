---
title: Some links, autolink all
---
Some links:

* https://www.bibo.com
* mailto--
* titled linK

Here is some other portion of a text containing a link mailto-- that will also be altered.
