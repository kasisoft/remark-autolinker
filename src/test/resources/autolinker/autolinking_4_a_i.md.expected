---
title: Some links, autolink all
---

Some links:

* https://www.bibo.com
* [mailto--](mailto://frog@insect.com)
* [titled linK](https://www.gollum.test)

Here is some other portion of a text containing a link [mailto--](mailto://frog@insect.com) that will also be altered.
